src/coreclr/jit/instr.cpp

```diff
@@ -523,6 +542,8 @@ void CodeGen::inst_RV_RV_RV(instruction ins,
 {
 #ifdef TARGET_ARM
     GetEmitter()->emitIns_R_R_R(ins, size, reg1, reg2, reg3, flags);
+#elif defined(TARGET_LOONGARCH64)
+    GetEmitter()->emitIns_R_R_R(ins, size, reg1, reg2, reg3);
```

Comment:
This is the same as the TARGET_XARCH case below

Reply:
OK, thanks.
I will move it below.